procmgr/main.cpp

```diff
@@ -2381,10 +2476,6 @@ void pingDeviceThread()
                                 }
                                 else
                                 {
-                                    processManager.distributeConfigFile("system");
```

Comment:
Between the previous change and this change, I don't see an unconditional call to distributeConfigFile().  Maybe it is called deeper in the call stack?